src/xcode/ENA/ENAUITests/TicketValidation/ENAUITests_19_TicketValidation.swift

```diff
@@ -17,8 +17,9 @@ class ENAUITests_19_TicketValidation: CWATestCase {
 		app.setLaunchArgument(LaunchArguments.onboarding.setCurrentOnboardingVersion, to: true)
 	}
 	
-	func test_ticketValidation_flow() throws {
+	func test_ticketValidation_flow_open() throws {
```

Comment:
Do we not need some screenshot for the open state?

Reply:
No, we don't